specification/paths/Brokers-broker_id-carriers.json

```diff
@@ -0,0 +1,109 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-broker_id"
+    }
+  ],
+  "get": {
+    "tags": [
+      "Brokers"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "enterprise.manage"
```

Comment:
Since it's a GET endpoint, should we allow `broker.member` to access it as well?

Reply:
I'm having a hard time knowing which scope is used in what scenario. If users with the `broker.member` scope have needs for viewing which carriers the broker uses then we should add it.

Reply:
We can also wait until the need is there. It's only used by admins for now.